spec/commands/v2/discard_draft_spec.rb

```diff
@@ -227,32 +230,32 @@
         it "deletes the draft for the given locale" do
           expect {
             described_class.call(payload)
-          }.to change(ContentItem, :count).by(-1)
+          }.to change(Edition, :count).by(-1)
 
-          expect(ContentItem.exists?(id: french_draft_item.id)).to eq(false),
-            "The French draft item was not removed"
+          expect(Edition.exists?(id: french_draft_item.id)).to eq(false),
+                                                               "The French draft item was not removed"
```

Comment:
a very small thing, but maybe we could avoid this (slightly hideous) indentation:

```
msg = "The French draft item was not removed"
expect(Edition.exists?(id: french_draft_item.id)).to eq(false), msg
```